js_built_in_functions.py

```diff
@@ -5,5 +5,16 @@
 generic AST string
 """
 class js_built_in_functions(enum.Enum):
+    #Array manipulation
     push = "appendStatement"
```

Comment:
Uber nit: Is it possible for the values of this enum to also be enums to avoid typo mistakes between the .js and .py file?

Reply:
Yes, but I thought that would make it less readable. Do you think I should still do it? @Danish21